app/javascript/components/SearchPanel.js

```diff
@@ -26,15 +26,15 @@ export const SearchContext = React.createContext(searchContexts.study);
  * This is the entry point into React code from the traditional JS code
  * See related integration at /app/javascript/packs/application.js
  */
-export default function SearchPanel() {
+export default function SearchPanel(props) {
   // Note: This might become  a Higher-Order Component (HOC).
   // This search component is currently specific to the "Studies" tab, but
   // could possibly also enable search for "Genes" and "Cells" tabs.
 
   return (
     <SearchContext.Provider value={searchContexts.study}>
       <div className='container-fluid' id='search-panel'>
-        <KeywordSearch />
+        <KeywordSearch updateKeyword={props.updateKeyword}/>
```

Comment:
this structure feels odd that you have to pass handlers down to keyword search, but not to facets panel.  I'd like the home page component to either handle all searches or none.

Reply:
We've discussed this. Further refinement needs to be made. However, this decision will be made as further components are developed.